### DISCUSSION TITLE
Thank You- Best Skin For My PWS

### DISCUSSION THREAD

**Discussion by gary-hammer** [Daveiano/weewx-wdc#148] [General]
I just wanted to say thanks for such a great skin.
With the addition of live data, this fills my need for a general website display and for my home tablets displays.
I was a longtime Belchertown user (years) and this skin replaced it within a day.

Excellent text, info, graphs and the presentation of the statistics are something I never knew I needed.

**Comment by Daveiano** [comment]
Thank you very much for these kind words! Very appreciated, thank you!
